Build/Remotion.Server.build

```diff
@@ -107,4 +107,10 @@
 
   <Import Project="$(CustomizationsDirectory)Server\*.targets" />
 
+  <Target Name="CheckBuildLocationProperties" >
+  </Target>
+
+  <Target Name="CheckBuildConfigurationProperties" >
+  </Target>
+
```

Comment:
Why is this needed?

Reply:
The target `CheckProperties` defined in the same file requires those targes. I was unsure if I should add these empty targets and leave them as an "extension point" or remove the dependency on them altogether

Reply:
I see. Is this the right issue to make this change? Is this generally even needed? I guess that's one for @MichaelKetting

Reply:
It literally wouldn't compile without them, so we either include these empty targets or remove the dependencies on them

Reply:
I don't understand. this worked before, the targets are defined in Properties.targets and loaded via Remotion.build import-startement. This should not be needed.

Reply:
You're right, I reverted the change and it still works. I must have done something else wrong before